gst/nnstreamer/tensor_common_pipeline.c

```diff
@@ -597,6 +597,7 @@ gst_tensor_aggregation_add_data (GHashTable * table, const guint32 key)
   else
     hashkey = key;
   aggr = g_new0 (gst_tensor_aggregation_data_s, 1);
+  g_assert (aggr != NULL);
```

Comment:
Don't fix this. Mark this "false positive". I have reported this issue to SE (SVACE maintainer) and it appears that they didn't fix it yet; g_new series are supposed to kill the process if g_new fails.

Reply:
Besides, try not to ass g_assert in non-test-case codes. Try to make proper error handling code instead.

Reply:
updated